packages/host/src/Host.tsx

```diff
@@ -30,30 +30,32 @@ const queryClient = new QueryClient({
 });
 
 const Host: FC = () => (
-  <React.Suspense fallback={<RandomLoader />}>
+  <React.Suspense fallback={<div />}>
```

Comment:
oh. heh. The loading spinners were suspending and had no fallback of their own 🙄 
I don't know if this is kosher or not, seems to work 🤷